pom.xml

```diff
@@ -2129,25 +2118,26 @@
         <spring4.version>4.3.20.RELEASE</spring4.version>
         <spring5.version>5.1.5.RELEASE</spring5.version>
         <surefire.version>3.0.0-M3</surefire.version>
-        <validation.impl.version>6.0.17.Final</validation.impl.version>
+        <validation.impl.version>7.0.0.Alpha1</validation.impl.version>
         <weld.version>2.2.14.Final</weld.version> <!-- 2.4.1 doesn't work - bv tests -->
-        <weld3.version>3.0.0.Final</weld3.version>
+        <weld3.version>4.0.0.Alpha1</weld3.version>
```

Comment:
I am afraid we need a CQ for this :(

Reply:
CQs are filled - references are in main comments